### ISSUE TITLE
Staging Review finding: `aria-labelledby` on "Review your information now" may block voice users.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#88999]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Modernized Check-in
**Product:** Pre-Check-in & eCheck-in
**Feature:** New Landing page design

## Findings details
**VA.gov Experience Standard - issue:** [User can't complete a necessary flow or hits a dead end.](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On /appointment-pre-check-in/appointments, the "Review your information now" link has an `aria-labelledby` that points to the H3 heading on the card. This announces the link as identical to the heading: "Review your contact information for your [day, month, date, year, time] appointment."

For sighted voice command users (eg. Dragon NaturallySpeaking on Windows, Apple Voice Control on iOS or Mac, etc.), the common interaction pattern goes:
- Read visible link text, "Review your information now"
- Speak command, "Click Review your information now"
- Link is clicked.

But voice command software looks for matches in the accessibility tree, where the `aria-labelledby` replaces the visible link text. When the link name in the accessibility tree is close to what's spoken, there's a decent chance of a partial match --- especially if what's spoken is word-for-word included in the name in the accessibility tree. But I think this is enough of a divergence here that most software wouldn't give you a match. (Tested with Dragon, was not able to get a match, and Dragon tends to be one of the better ones for this.)

There are usually other mechanisms available for clicking a link for voice command users, but this is a point of friction that should be easy to smooth out.

<details><summary>Link, screenshot or steps to recreate</summary>

![Card with heading "Review your contact information for your Saturday, July 20 8:30 a.m. appointment" and visible link text of "Review your information now."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/Brian-53488-1.png?raw=true)

</details>

## Recommended action

It's unclear from testing if users will ever have more than one appointment listed here. If not, no ARIA is needed --- if there's only ever one "Review your information now" link, no need to add anything extra to differentiate it from other identical links.

If you can have more than one appointment (which I'm assuming is true?) then I would recommend something like this instead:
```
<a data-testid="action-link" class="vads-c-action-link--green" href="/" aria-label="Review your information now for Saturday, July 20 8:30 a.m. appointment">Review your information now</a>
```
Or something similar, that uses date/time to differentiate it from other appointments. If appointments are always on the same day, you can probably drop the date and just do time.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by loripusey** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Check-in-Experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/88999?planning-poker) with Zenhub @brianseek

**Comment by benbrasso-agile6** [comment]
Screenshot being silly and not loading for me

**Comment by brianseek** [comment]
@benbrasso-agile6 try again, I think it was zenhub shenanigans

**Comment by benbrasso-agile6** [comment]
If I'm understanding the ticket, that's a screenshot for multiple appts for pre check in payload and therefore LGTM.

**Comment by loripusey** [comment]
LGTM closing
